.github/workflows/release-image.yaml

```diff
@@ -0,0 +1,38 @@
+name: Generate Release Image
+
+on: [pull_request]
+
+jobs:
+  release-image:
+    # Only generate the release image from the PR contents if the PR was in the
+    # monorepo and opened by lit-robot.
+    if: github.repository == 'lit/lit' && github.actor == 'lit-robot'
```

Comment:
No action needed.

We use the same `github.actor` check for not running the lint & changeset workflow step for the "Version Packages" PR. I've noticed that when another changeset workflow is triggered while a "Version Packages" PR is open, the workflow updates the existing PR but "github-actions" becomes the actor (even though the commit is authored by lit-robot) due to the credentials used to force push and update the PR.

So I believe this workflow will only run on the initial PR creation. Updates must happen by closing that PR and re-opening. We currently already do this because workflows sometimes don't run when the PR is updated anyway so this will work for us as is.

Reply:
Noting that this was a brilliant comment - and resulted in a change in approach that is better. Thank you!